src/SignalR/clients/java/signalr/src/test/java/com/microsoft/signalr/WebSocketTransportTest.java

```diff
@@ -17,11 +17,11 @@
 
 class WebSocketTransportTest {
     // @Test Skipping until we add functional test support
-    public void WebSocketThrowsIfItCantConnect() {
-        Transport transport = new WebSocketTransport(new HashMap<>(), new DefaultHttpClient());
-        RuntimeException exception = assertThrows(RuntimeException.class, () -> transport.start("http://url.fake.example").blockingAwait(1, TimeUnit.SECONDS));
-        assertEquals("There was an error starting the WebSocket transport.", exception.getMessage());
-    }
+    // public void WebSocketThrowsIfItCantConnect() {
+    //     Transport transport = new WebSocketTransport(new HashMap<>(), new DefaultHttpClient());
+    //     RuntimeException exception = assertThrows(RuntimeException.class, () -> transport.start("http://url.fake.example").blockingAwait(1, TimeUnit.SECONDS));
+    //     assertEquals("There was an error starting the WebSocket transport.", exception.getMessage());
+    // }
```

Comment:
Is commenting out just the `@Test` attribute no longer sufficient?

Reply:
I removed the DefaultHttpClient constructor that was being used. Maybe I should just delete the test

Reply:
That or just call the new constructor with a no-op builder. Up to you.